_posts/2024-11-08-navigating-the-ecosystem.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  "Finding your way around the CHERIoT ecosystem"
+date:   2024-11-08
+categories: rtos sonata git
+author: Phil Day
+---
+With a new set of folks in the DSbD cohort 6 just getting started with their Sonata boards I thought it might be useful to capture some of the things I've learn in the last few months about the ecosystem.
```

Comment:
```suggestion
With a new set of folks in the DSbD cohort 6 just getting started with their Sonata boards I thought it might be useful to capture some of the things I've learnt in the last few months about the ecosystem.
```